src/components/action-popover/action-popover.stories.tsx

```diff
@@ -436,7 +436,7 @@ KeyboardNavigationLeftAlignedSubmenu.storyName =
 export const KeyboardNavigationRightAlignedSubmenu: Story = () => {
   return (
     <Box height={250}>
-      <ActionPopover>
+      <ActionPopover ml={0}>
```

Comment:
**nitpick(non-blocking)**: I think you might also need to add `submenuPosition="right"` prop here too.

Reply:
Forgot to add the screenshot. This is how it currently looks in the `docs` page on your PR. 
![Screenshot 2024-04-16 at 16 18 14](https://github.com/Sage/carbon/assets/56251247/ad820090-2dea-4533-92ef-f2d3dcda15f2)

Reply:
Hi,

The `docs` page has been fixed.

![Screenshot 2024-05-09 084858](https://github.com/Sage/carbon/assets/101639841/8f14ac6a-c09b-4275-829b-162330fe49d4)